### PR TITLE
Make german translation more gender neutral #4755

### PR DESCRIPTION
**Pull request by ghost**
As requested in issue #4755 a more gender neutral form of the german translation.

I preferred the "Profil" form over several genderspeak forms, because: 

- it doesn't hinder the flow while reading
- it doesn't confuse screen readers (for visually impaired)
- it does even cover the situation where the entity behind a profile is a just a script

### CODE REVIEW

**Review by Gargron — APPROVED** [review]

File: config/locales/simple_form.de.yml [hunk 1]
```diff
@@ -33,10 +33,10 @@ de:
         setting_default_privacy: Beitragsprivatspäre
         severity: Gewichtung
         type: Importtyp
-        username: Nutzername
+        username: Profilname
       interactions:
         must_be_follower: Benachrichtigungen von Nicht-Folgern blockieren
```
Comment (averageunicorn) [inline config/locales/simple_form.de.yml]: "Nicht-Folgern" should be changed to "Nicht-Folgenden"

**Comment by averageunicorn** [comment]
Hi, thanks for this (not for calling the forms generally preferred by nonbinary people "genderspeak", though). One thing that I noticed that still has to be changed:

in simple_form.de.yml must_be_follower: "Nicht-Folgern" to "Nicht-Folgenden"

**Comment by andydrop** [comment]
@averageunicorn 
The patch just covered the appearences of *utzer in mastodon. I will do a more broad approach within the next week, but that will need a quiet moment, that i don't have right now.

The focus will remain on screen reader usability and neutral forms combined.

Sorry for the "genderspeak" word. I used it in lack of another single word to subsume more than a dozen different approaches of gender neutral language that use syntax to specify diversity.

I would like to get more hints to other "words to change" via @DrOps@mastodon.social for the broader patch

**Comment by averageunicorn** [comment]
@andydrop 
Great, thanks for exaplaining. If anything turns out to be awkward to rephrase, -lon was another form that was suggested for its screenreader compatibility. And the neopronouns (such as xier or nin) will work as well.

**Comment by andydrop** [comment]
@averageunicorn 
If i absolutely had to use one of these forms, I personally like the Binnen-I (NutzerIn) and am appealed by the -lon (because it reminds me of Zini the Wuslon, for whom, I think, no gender was stated ever)

But generally i admire the "Marina Weisband" style of gender neutral writing.

**Pull request merged**